lib/common/client/concerns/mhv_locked_session_client.rb

```diff
@@ -19,8 +19,8 @@ module MhvLockedSessionClient
         extend ActiveSupport::Concern
         include SentryLogging
 
-        LOCK_RETRY_DELAY = 1 # Number of seconds to wait between attempts to acquire a session lock
-        RETRY_ATTEMPTS = 10 # How many times to attempt to acquire a session lock
+        LOCK_RETRY_DELAY = 0.3 # Number of seconds to wait between attempts to acquire a session lock
+        RETRY_ATTEMPTS = 40 # How many times to attempt await of acquiring a session lock by a preceding request
```

Comment:
Per my understanding - there should not be. Although the 10 sec retry period is arbitrary, we did not see any issues while this was in place for MHV Medical Records api services. as for the `each_ttl` value, we did bump up the retry period to exceed `ttl` deliberately with the idea that another thread may reacquire a lock in case of a timeout. @mmoyer-va I did not misrepresent that, did I?

Reply:
That's correct -- we deliberately want the thread to try for longer than the thread can be locked, so that it has a chance to acquire a stale lock if possible.

Under normal circumstances, the thread will not try to acquire a lock for the full 10-12 seconds. Normally, the first thread will create a session in about 1 second and release the lock, and any threads that are polling will immediately have that session available for use.